spec/index.bs

```diff
@@ -2692,13 +2694,13 @@ has a default layout created and used instead.
 
             1. If |groupDescs|[|group|] has an entry |previousEntry| with {{GPUBindGroupLayoutEntry/binding}} equal to |binding|:
 
-                1. If |previousEntry| is equal to |entry| up to {{GPUBindGroupLayoutEntry/visibility}}:
+                1. If a member other than {{GPUBindGroupLayoutEntry/visibility}} or {{GPUBindGroupLayoutEntry/minBufferBindingSize}} is unequal between |previousEntry| and |entry|:
```

Comment:
the `minBufferBindingSize` should be updated to be the max(`minBufferBindingSize` between stages)

Reply:
We set the previousEntry's minBufferBindingSize to max(entry, previousEntry), wouldn't this give the max across stages in the final entry since we loop over all stages? Or should we state it some other way?